jupyter_bifrost/bifrost.py

```diff
@@ -185,7 +185,8 @@ def create_graph_data(
                 "width": 550,
                 "height": 405,
                 "mark": kind,
-                "params": [{"name": "brush", "select": "interval"}],
+                # "params": [{"name": "brush", "select": "interval"}],
+                "params": [{"name": "grid", "select": "interval", "bind": "scales"}],
```

Comment:
Yeah if we can toggle these back and forth with a button or keyboard command that'd be great!

Reply:
Added a button to do this!